AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -39,6 +39,9 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 ##.widget_blog_subscription
 ~theoatmeal.com,~copays.org,~nothing.tech,~mega-debrid.eu,~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
 !
+fortune.com#$#.preloaded_lightbox { display: none!important; }
+fortune.com#$#.fb_lightbox-overlay { display: none!important; }
+fortune.com#$#body { overflow: auto!important; }
```

Comment:
I don't see this on my end. Can you reproduce it?

Reply:
@zloyden Yes.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/35523759/178242641-5f3e69c5-75eb-40d0-b631-09ac54270a04.png)
</details>